python/deltalake/table.py

```diff
@@ -247,7 +248,7 @@ def __init__(
                                 This can decrease latency if there are many files in the log since the last checkpoint,
                                 but will also increase memory usage. Possible rate limits of the storage backend should
                                 also be considered for optimal performance. Defaults to 4 * number of cpus.
-
+        :param lazy_load: when true the table metadata isn't loaded
         """
         self._storage_options = storage_options
         self._table = RawDeltaTable(
```

Comment:
I think something went wrong here, since you're loading it first with everything